test/commentRouterTest.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+require('dotenv').load();
+const superagent = require('superagent');
+const storage = require('../lib/storage');
+const expect = require('chai').expect;
+const Comment = require('../model/comment.js');
+const baseUrl = `http://localhost:${process.env.PORT || 3000}`;
+require('../server.js');
+
+describe('testing comment router', function(){
+  describe('testing POST /api/comments', function(){
+    after((done) => {
+      Comment.deleteById(this.tempComment.id)
+      .then(() => done())
+      .catch(done);
+    });
+
+    it('should create comment', (done) => {
+      superagent.post(`${baseUrl}/api/comments`)
+      .send({
+        announcer: 'Troy Aikman',
+        comment: 'He did a thing',
+        year: '2017',
+      })
+      .then(res => {
+        console.log(res.body);
+        console.log(res.statusCode);
+        expect(res.status).to.equal(200);
+        expect(res.body.announcer).to.equal('Troy Aikman');
+        expect(res.body.comment).to.equal('He did a thing');
+        expect(res.body.year).to.equal('2017');
+        expect(Boolean(res.body.id)).to.equal(true);
+        this.tempComment = res.body;
+        done();
+      })
+      .catch(done);
+    });
+
+    it('should return 400 status code', (done) => {
+      superagent.post(`${baseUrl}/api/comments`)
+      .send({
+        announcer: 'Troy Aikman',
+      })
+      .then(done)
+      .catch(err=> {
+        console.log(err.status);
+        expect(err.status).to.equal(400);
+        done();
+      })
+      .catch(done);
+    });
+  });
+});
+
+describe('testing DELETE aka thats so fetch', function(){
```

Comment:
I just wrote a test for you to test out your DELETE route myself. Replace your it block with this. 

```
    it('should delete a comment', (done) => {
      superagent.delete(`${baseUrl}/api/comments/${this.tempComment.id}`)
      .then((res) => {
        expect(res.status).to.equal(204);
        done();
      })
      .catch(done);
    });
```